scripts/generate-rules-table.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+// Automatically regenerates the rules table in readme.md.
+
+const fs = require('fs');
+const path = require('path');
+const {rules, configs} = require('../');
+
+const pathReadme = path.resolve(__dirname, '../readme.md');
+const readmeContents = fs.readFileSync(pathReadme, 'utf8');
+const tablePlaceholder = /<!--RULES_TABLE_START-->[\S\s]*<!--RULES_TABLE_END-->/;
```

Comment:
Will `dotAll`flag work?

Reply:
What do you mean by that?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/dotAll

Reply:
Updated to use `RegExp(..., 's')`.